challenges/cat-photo-gallery/script.js

```diff
@@ -0,0 +1,36 @@
+window.onload = getRandomCatPicture ()
+
+async function getRandomCatPicture () {
+    await fetch("https://cataas.com/cat?json=true")
+    .then(response => response.json())
+    .then(randomCat => {
+        const image = document.querySelector("img");
+        return image.src = "https://cataas.com" + randomCat.url;
```

Comment:
you can't assign and return a value / object at the same time.
if you method called `getRandomCatPicture()` you supposed to return an image from it:
```js
const image = document.querySelector("img");
image.src = "https://cataas.com/" + randomCat.url;
return image;
```
or maybe an URL to an image: 
```js
return "https://cataas.com/" + randomCat.url;
```

Reply:
if you change something in you method/function, better call it `setRandomCatPicture()`:
```js
const image = document.querySelector("img");
image.src = "https://cataas.com/" + randomCat.url;
```
without returning anything from it.

Reply:
additionally, 
`"https://cataas.com" + randomCat.whatever` won't work
the correct link should be:
`"https://cataas.com/cat/" + randomCat.whatever`

__Note__:  replace `whatever` with the required thing.